mediainfo/src/main/cpp/mediainfo.cpp

```diff
@@ -166,6 +184,12 @@ void media_info_build(JNIEnv *env, jobject jMediaInfoBuilder, const char *uri) {
                 break;
         }
     }
+
+    for (int pos = 0; pos < avFormatContext->nb_chapters; pos++) {
+        onChapterFound(env, jMediaInfoBuilder, avFormatContext, pos);
+    }
+
+    avformat_close_input(&avFormatContext);
```

Comment:
avformat context is only released after `MediaInfo.release()` is called. 
Avformat release called here:-  https://github.com/anilbeesetti/nextlib/blob/2d7d78d8f3ffd7c9dda4f99e09be3302ac5fd01d/mediainfo/src/main/cpp/frame_loader_context.cpp#L15